resources/assets/lib/utils/beatmap-helper.ts

```diff
@@ -69,6 +70,15 @@ export function getDiffRating(rating: number) {
   return 'expert-plus';
 }
 
+// TODO: should make a Beatmapset proxy object or something
+export function getTitle(beatmapset: BeatmapsetJson) {
```

Comment:
php side needs this as well for (at least) `beatmapsets.show` and `scores.show` (html) page title.